### DISCUSSION TITLE
It is recommended to compile the code into dart

### DISCUSSION THREAD

**Discussion by yibo7** [CoderLine/alphaTab#630] [Ideas]
Since typescript can be compiled into C #, why not compile it into dart? At present, flutter has been running stably on various mobile platforms

**Comment by Danielku15** [comment]
TypeScript cannot be compiled into C# by default. 😉  I made a dedicated decision that I want to provide alphaTab for the .net platform and developed on my own a toolchain which allows me to compile the alphaTab TypeScript codebase to C# to ship a .net version. This doesn't come easily. 

I currently have no interest into providing a "Dart" version of alphaTab. I am focusing on targeting Android and iOS through a toolchain targeting Kotlin. If you have a strong interest into having a Flutter version, feel free to describe in further extend why this would be beneficial for you and maybe others.

**Reply by yibo7** [reply]
Thanks, as long as we can solve Android and IOS platform code, I can load it with flutter plug-in
